bin/build-version-json.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Why is this copied? Can't we reuse `.utils/build-version-json.sh`?

Reply:
I am probably going to change some things here.   would you like if the GitHub run number include ?  it's a 

"unique number for each run of a particular workflow in a repository. This number begins at 1 for the workflow's first run, and increments with each new run. This number does not change if you re-run the workflow run."

Reply:
sorry, include what? The unique number? Like we have, e.g. [here](https://addons.mozilla.org/__frontend_version__) for AMO frontend (`build`)?

Reply:
Yeah it would be similar to that.

Reply:
Oh ok, I guess if this could be a link that'd be great. If it's just a numeric ID, I guess that's better than nothing. What matters the most for me is to have the git tag or git hash in this `version.json` file.